routes/shop.js

```diff
@@ -36,6 +36,16 @@ const shopItems = [
 
         }
     },
+    {
+        name: "Birthday Change",
+        desc: "Change your birthday once per purchase",
+        key: "bdayChange",
+        price: 10,

```

Comment:
I personally feel that birthday does not need to be a purchase but this works too xD

Reply:
- For future consideration -